### PR TITLE
WWST-1481 leedarson dim bulb

### PR DESCRIPTION
**Pull request by wen33wenS**
Add a new DTH for leedarson dim bulb

### CODE REVIEW

File: devicetypes/smartthings/leedarson-dim-bulb.src/leedarson-dim-bulb.groovy [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+        capability "Actuator"
+        capability "Configuration"
+        capability "Refresh"
+        capability "Switch"
+        capability "Switch Level"
+        capability "Health Check"
+        capability "Light"
+
+        fingerprint profileId: "0104", inClusters: "0000, 0004, 0003, 0006, 0008, 0005, FFFF, 1000", outClusters: "0019", model: "ZBT-DIMLight-GLS0000", manufacturer: "LDS", deviceJoinName: "Bulb"
```
Comment (tpmanley) [inline devicetypes/smartthings/leedarson-dim-bulb.src/leedarson-dim-bulb.groovy]: This looks like a standard dimmable bulb so you can use the Zigbee Dimmer DTH and don't need to create a new one.

**Comment by wen33wenS** [comment]
@tpmanley @greens @varzac plz help to review leedarson dim bulb DTH. Thanks.

File: devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy [hunk 2]
```diff
@@ -45,6 +45,8 @@ metadata {
         fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, 0B05, FC01", outClusters: "0003, 0019", manufacturer: "LEDVANCE", model: "BR30 W 10 year", deviceJoinName: "SYLVANIA Smart 10Y BR30 Soft White"
         fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, 0B05, FC01", outClusters: "0003, 0019", manufacturer: "LEDVANCE", model: "PAR38 W 10 year", deviceJoinName: "SYLVANIA Smart 10Y PAR38 Soft White"
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008", outClusters: "0003, 0006, 0008, 0019, 0406", manufacturer: "Leviton", model: "ZSD07", deviceJoinName: "Leviton Lumina RF 0-10V Dimming Wall Switch"
+        fingerprint profileId: "0104", inClusters: "0000, 0004, 0003, 0006, 0008, 0005, FFFF, 1000", outClusters: "0019", model: "ZBT-DIMLight-GLS0000", manufacturer: "LDS", deviceJoinName: "Bulb"
```
Comment (tpmanley) [inline devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy]: Can you use a more specific join name? Also does this device really have a FFFF cluster?

Comment (tpmanley) [reply devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy]: Put the manufacturer first to match the other fingerprints

**Comment by wen33wenS** [comment]
bulb
FFFF is read from hub, refer to jpg
The marketing team has not confirm the product name.

**Review by tpmanley — APPROVED** [review]

**Pull request merged**